i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,59 @@
+from torch import nn
+
+
+class ConformerConvolutionV1(nn.Module):
+    """
+    Conformer convolution module.
+    see also: https://github.com/espnet/espnet/blob/713e784c0815ebba2053131307db5f00af5159ea/espnet/nets/pytorch_backend/conformer/convolution.py#L13
+    """
+
+    def __init__(self, channels, kernel_size, dropout=0.1):
+        """
+        :param int channels: number of channels for conv layers
+        :param int kernel_size: kernel size of conv layers
+        """
+        super().__init__()
+
+        # kernel size has to be odd to get same input length when using odd strides. here we use stride 1.
+        assert (kernel_size - 1) % 2 == 0
+
+        self.pointwise_conv1 = nn.Conv1d(
+            in_channels=channels,
+            out_channels=2 * channels,
+            kernel_size=1,
+        )
+        self.depthwise_conv = nn.Conv1d(
+            in_channels=channels,
+            out_channels=channels,
+            kernel_size=kernel_size,
+            padding=(kernel_size - 1) // 2,
```

Comment:
Why not just `padding = "same"`? That might also solve the problem from the comment above about odd kernel sizes?

Reply:
I tried that and it did not work. I also get this warning msg:
```
Warning from pytorch:
Using padding='same' with even kernel lengths and odd dilation may require a zero-padded copy of
the input be created.
```

Reply:
This is just a warning about performance. It says that the performance might be suboptimal in this case, as it requires an additional copy. It should still work though.

Reply:
If padding="same" is really not supported in ONNX, as noted in `https://stackoverflow.com/questions/68565147/pytorch-unable-to-export-trained-model-as-onnx` we should use the integer version here.